README.md

```diff
@@ -53,17 +181,219 @@ const createMyFirstNFT = async () => {
+
+```javascript
+const create createNFTManually = async () => {
+  try {
+    // STEP 3.1 : Here we submit the transaction hex value. 
+    // Here no callback function is used but the second parameter can be a callback function that help you to handle the result.
+    // Once again, we use here the tx hex signedTxHash, from the previous step.
+    const submitTxHash = await submitTxHex(signTxHash)
+    // Do something with the final tx hash. 
+    console.log(submitTxHash);
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
++ `submitTxBlocking()` - The most most convenient way to get Events and datas : This function will sign the transaction for you if you pass a keyring (one less thing to worry about) and it's blocking the execution flow until the transaction is either in a block or in a finalized block. Since submitting needs to work will all kinds of transactions, the result is an object that contains all the events that have happen (instead of only specific ones).
+Note: Here you do not need to necessarily pass a signed `txHash`. If you pass the Keyring as a parameter and an unsigned tx hash as the signing process will be done here for you.
+
+```javascript
+const create createNFTManually = async () => {
+  try {
+    ...
+
+    // STEP 3.2 : Here we submit the transaction hex value signed before. 
+    // We could have used the unsigned tx Hash and pass a keyring as a third parameter instead.
+    // Once again, we use here the tx hex signTxHash, from the previous step.
+    const submitTxHash = await submitTxBlocking(signTxHash, WaitUntil.BlockInclusion)
+
+    // Do something with the events recieved. 
+    console.log(submitTxHash.findEvents(NFTCreatedEvent));
+    ...
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
++ `submitTxNonBlocking()`: This one works as the `submitTxBlocking` but in a non blocking way. It returns a pair of objects that are used to track the progress of the transaction execution. The first returned object is a conditional variable which can yield the information if the operation is finished. The second returned objects is an array of events which gets populated automatically once the operation is finished.
+
+```javascript
+const create createNFTManually = async () => {
+  try {
+    ...
+
+    // STEP 3.3 : It get the same parameters as the submitTxBlocking
+    // Here we submit the transaction hex value signed before. 
+    // We could have used the unsigned tx Hash and pass a keyring as a third parameter instead.
+    // Once again, we use here the tx hex signTxHash, from the previous step 2.
+    const submitTxHash = await submitTxNonBlocking(signTxHash, WaitUntil.BlockInclusion)
+
+    // Do something with the events recieved. 
+    console.log(submitTxHash);
+    ...
+  } catch (e) {
+    console.log(e)
+  }
+}
+```
+
+**Ternoa Token Standardization** : Awaiting finalised TRC versions (Ternoa's Request for Comment)
+
+## API Architecture : 
```

Comment:
It is not an API!! 😱
We should say `Code Architecture` instead 🙂

Reply:
Oof , my bad (fixed)

Looks like caffeine and I don't mix well